docs/contributing/testing.md

```diff
@@ -65,27 +65,36 @@ You should test that your contribution works:
 
 ## 5. Run the automated tests
 
+Automated testing helps ensure that our code is as free of bugs as it can be. It also lets us know immediately if a change we make breaks any other part of the code.
```

Comment:
Should we add how it lets us know? Perhaps - It also provides immediate notifications if a change we make...etc
or
It also immediately displays where a change we make...etc